src/cogent3/app/composable.py

```diff
@@ -130,6 +130,22 @@ def to_json(self):
         return json.dumps(self.to_rich_dict())
 
 
+MODEL_RESULT_TYPE = "model_result"
```

Comment:
Move all the `<something>_RESULT_TYPE` to the bottom of this block of definitions. Separate them by a blank line and make alphabetical.